front-end-admin/src/components/need/NeedInfo.vue

```diff
@@ -2,48 +2,74 @@
-          <label class="control-label col-sm-2">Full Name:</label>
-          <input name="need_name" type="text" width="50" value="{{need_name}}">
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-2">Description:</label>
-          <textarea rows="4" cols="50" name="need_description" form="needform">need_description</textarea>
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-2">Need URL:</label>
-          <input name="need_url" type="text" width="50" value="{{ need_url }}">
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-2">Quantity:</label>
-          <input id="need_quantity" name="need_quantity" type="number" min="1" step="1" value ="{{need_quantity}}"/>
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-2">Units:</label>
-          <input name="need_unit" type="text" width="50" value="{{ need_unit }}">
-        </div>
-        <div class="form-group">
-          <label class="control-label col-sm-4>">Associated Callout: </label>
-          <a href="callouts/{{callout_id}}">link</a>
+       <div class="container">
+        <h2>Need Info Page for {{ need.name }}</h2>
+        <form class="form-horizontal" action="/updateneed" method="POST" id="needform">
+          <button type="submit">Update Need</button>
+          <input type="hidden" name="need_id" v-model="need.id">
           <div class="form-group">
-            <label class="control-label col-sm-4>">Status: </label>
-            <input type="checkbox" name="active" checked="{{active}}">Active?
+            <label class="control-label col-sm-2">Full Name:</label>
+            <input name="need_name" type="text" width="100" v-model="need.name">
           </div>
-          <button type="submit">Update Need</button>
-        </div>
-      </form>
+          <div class="form-group">
+            <label class="control-label col-sm-2">Description:</label>
+            <textarea rows="4" cols="50" v-model="need.description" form="needform">need_description</textarea>
+          </div>
+          <div class="form-group">
+            <label class="control-label col-sm-2">Need URL:</label>
+            <input name="need_url" type="text" width="50" v-model="need.url">
+          </div>
+          <!-- <div class="form-group">
+            <label class="control-label col-sm-2">Quantity:</label>
+            <input id="need_quantity" name="need_quantity" type="number" min="1" step="1" v-model="need_quantity"/>
+          </div> -->
+          <div class="form-group">
+            <label class="control-label col-sm-2">Units:</label>
+            <input name="need_unit" type="text" width="50" v-model="need.unit">
+          </div>
+          <div class="form-group">
+            <label class="control-label col-sm-4>">Associated Callout: </label>
+            <!-- <a href="callouts/{{callout_id}}">link</a> -->
```

Comment:
What does this refer to in a need? I think this can removed because it was from the old database